src/sentry/api/endpoints/project_agnostic_rule_conditions.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+
+from sentry.api.base import Endpoint
+from sentry.rules import rules
+from rest_framework.response import Response
+
+
+class ProjectAgnosticRuleConditionsEndpoint(Endpoint):
+    permission_classes = ()
```

Comment:
Should all roles be able to use this endpoint?

Reply:
Should be. It contains no project or org specific info

Reply:
Work for backend shifted to this PR: https://github.com/getsentry/sentry/pull/16933